[2023-09-13T07:38:28.189+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: weather_data.extract_data scheduled__2023-08-21T00:00:00+00:00 [queued]>
[2023-09-13T07:38:28.239+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: weather_data.extract_data scheduled__2023-08-21T00:00:00+00:00 [queued]>
[2023-09-13T07:38:28.245+0000] {taskinstance.py:1308} INFO - Starting attempt 1 of 1
[2023-09-13T07:38:28.303+0000] {taskinstance.py:1327} INFO - Executing <Task(PythonOperator): extract_data> on 2023-08-21 00:00:00+00:00
[2023-09-13T07:38:28.316+0000] {standard_task_runner.py:57} INFO - Started process 88 to run task
[2023-09-13T07:38:28.372+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'weather_data', 'extract_data', 'scheduled__2023-08-21T00:00:00+00:00', '--job-id', '26', '--raw', '--subdir', 'DAGS_FOLDER/weather_dag.py', '--cfg-path', '/tmp/tmpnmen8xzc']
[2023-09-13T07:38:28.382+0000] {standard_task_runner.py:85} INFO - Job 26: Subtask extract_data
[2023-09-13T07:38:28.510+0000] {task_command.py:410} INFO - Running <TaskInstance: weather_data.extract_data scheduled__2023-08-21T00:00:00+00:00 [running]> on host 190733975a8f
[2023-09-13T07:38:28.766+0000] {taskinstance.py:1547} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='weather_data' AIRFLOW_CTX_TASK_ID='extract_data' AIRFLOW_CTX_EXECUTION_DATE='2023-08-21T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-08-21T00:00:00+00:00'
[2023-09-13T07:38:29.141+0000] {python.py:183} INFO - Done. Returned value was: None
[2023-09-13T07:38:29.158+0000] {taskinstance.py:1350} INFO - Marking task as SUCCESS. dag_id=weather_data, task_id=extract_data, execution_date=20230821T000000, start_date=20230913T073828, end_date=20230913T073829
[2023-09-13T07:38:29.216+0000] {local_task_job_runner.py:225} INFO - Task exited with return code 0
[2023-09-13T07:38:29.254+0000] {taskinstance.py:2653} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2023-09-15T11:31:07.411+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: weather_data.extract_data scheduled__2023-08-21T00:00:00+00:00 [queued]>
[2023-09-15T11:31:07.440+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: weather_data.extract_data scheduled__2023-08-21T00:00:00+00:00 [queued]>
[2023-09-15T11:31:07.441+0000] {taskinstance.py:1308} INFO - Starting attempt 1 of 1
[2023-09-15T11:31:07.554+0000] {taskinstance.py:1327} INFO - Executing <Task(PythonOperator): extract_data> on 2023-08-21 00:00:00+00:00
[2023-09-15T11:31:07.566+0000] {standard_task_runner.py:57} INFO - Started process 327 to run task
[2023-09-15T11:31:07.575+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'weather_data', 'extract_data', 'scheduled__2023-08-21T00:00:00+00:00', '--job-id', '10', '--raw', '--subdir', 'DAGS_FOLDER/weather_dag.py', '--cfg-path', '/tmp/tmpxy9ny_gz']
[2023-09-15T11:31:07.579+0000] {standard_task_runner.py:85} INFO - Job 10: Subtask extract_data
[2023-09-15T11:31:07.782+0000] {task_command.py:410} INFO - Running <TaskInstance: weather_data.extract_data scheduled__2023-08-21T00:00:00+00:00 [running]> on host 4af8904e631c
[2023-09-15T11:31:08.118+0000] {taskinstance.py:1547} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='weather_data' AIRFLOW_CTX_TASK_ID='extract_data' AIRFLOW_CTX_EXECUTION_DATE='2023-08-21T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-08-21T00:00:00+00:00'
[2023-09-15T11:31:08.464+0000] {python.py:183} INFO - Done. Returned value was: None
[2023-09-15T11:31:08.509+0000] {taskinstance.py:1350} INFO - Marking task as SUCCESS. dag_id=weather_data, task_id=extract_data, execution_date=20230821T000000, start_date=20230915T113107, end_date=20230915T113108
[2023-09-15T11:31:08.574+0000] {local_task_job_runner.py:225} INFO - Task exited with return code 0
[2023-09-15T11:31:08.599+0000] {taskinstance.py:2653} INFO - 0 downstream tasks scheduled from follow-on schedule check
